packages/app-store/slack/zod.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from "zod";
+
+export const appDataSchema = z.object({});
+
+export const appKeysSchema = z.object({
+  client_id: z.string().min(1),
+  client_secret: z.string().min(1),
+});
+
+export const ZSlackCredentialKey = z.object({
```

Comment:
let's have the link to the relevant Slack docs for the object structure here and remove the commented out code.